src/index.html

```diff
@@ -11,12 +11,101 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          alt="Logo Moyo"
+          src="images/logo.png"
+          class="logo-header"
+        />
+      </a>
+      <nav class="menu">
+        <ul class="list-menu">
```

Comment:
Wrong BEM naming.

I see you have declared all your elements in the format "element-block" when it should actually be "block__element".
You should check the declaration for all your classes from here onwards. Here is an example:
```
<nav class="header__menu menu">
    <ul class="menu__list">
        <li class="menu__item">
            <a class="menu__link">
                Link
            </a>
        </li>
    </ul>
</nav>
```